### ISSUE TITLE
AVT3: Graphic name is not correct

### ISSUE THREAD

**Issue by xiepingp** [ibm-cloud-docs/Monitoring-with-Sysdig#12]
=====ENVIRONMENT=====
IBM CI 162 Checkpoint Violation: IBM 4.1.2 Name, Role, Value
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/name_role_value.html
Test Tool Type: JAWS Screen Reader 2019

=====DEFECT DESCRIPTION=====
Graphic names are not correct in Managing panels page

=====STEPS TO REPRODUCE=====
Keep JAWS running
-> Go to: https://cloud.ibm.com/docs/Monitoring-with-Sysdig?topic=Monitoring-with-Sysdig-panels
-> Click Insert + F3 to open the Virtual HTML Features dialog
-> Click down arrow to move to Graphics list, click OK button
![Sysdig - doc - graphic name 1](https://user-images.githubusercontent.com/63960906/91825662-22f18500-ec6f-11ea-9bc5-8efda3f22825.png)
![Sysdig - doc - graphic name 2 ](https://user-images.githubusercontent.com/63960906/91825677-271da280-ec6f-11ea-9730-f4e4e229cbc6.png)

===== EXPECTED RESULT =====
The intent of this Success Criterion is to ensure that Assistive Technologies (AT) can gather information about, activate(or set) and keep up to date on the status of user interface controls in the content.
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Comment by xiepingp** [comment]
@LOPEZDSR, would you please take a look at the issue? Thank you!

**Comment by xiepingp** [comment]
Retest the issue, all looks good:
![12 - fixed - 1](https://user-images.githubusercontent.com/63960906/98194371-31ec0080-1f5a-11eb-817c-bee1aeee36f7.png)
![12 - fixed - 2](https://user-images.githubusercontent.com/63960906/98194377-37e1e180-1f5a-11eb-85fa-16ee9cc95811.png)
Thanks for the fixing.
